foreman/foreman.spec

```diff
@@ -625,7 +691,7 @@ Summary: Foreman sqlite support
 Group:  Applications/System
 Requires: %{?scl_prefix_ror}rubygem(sqlite3) >= 1.3.6
 Requires: %{?scl_prefix_ror}rubygem(sqlite3) < 1.4.0
-Requires: %{name} = %{version}-%{release}
+Requires: %{name}-core = %{version}-%{release}
```

Comment:
All of these database subpackages ought to depend on `%{name}` as installing foreman-sqlite should work to install the entire application. Certainly the installer does this and the manual may suggest it too.

This'll conflict with foreman-plugin though which depends on a database and is the place you're trying to break the circular dependency...